### PR TITLE
Sell screen updates

### PR DESCRIPTION
**Pull request by lmillet3804**

### CODE REVIEW

File: app/sell/index.js [hunk 1]
```diff
@@ -47,6 +48,14 @@ const Sell = () => {
 
     const profileData = useSelector(selectProfileData)
 
+    const [refreshing, setRefreshing] = useState(false)
+
+    const onRefresh = useCallback(() => {
```
Comment (TheGoldenPatrik1) [inline app/sell/index.js:59]: Would it be possible to make the refresh event also reset the 60 second timer?

File: app/sell/success.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    const returnHome = () => {
+        clearRouterStack('/', navigation)
+    }
+    return (
+        <Page header="Success!">
+            <View style={styles.success}>
+                <Text style={styles.text}>
+                    Thank you for using MealMatch. Your funds will be released
+                    in the next 2 hours if there are no buyer complaints. {'\n'}
```
Comment (TheGoldenPatrik1) [inline app/sell/success.js]: Let's change this wording to "Thank you for using MealMatch. If there are no buyer complaints in the next two hours, your funds will be released sometime after that."

**Review by TheGoldenPatrik1 — CHANGES_REQUESTED** [review]

File: app/sell/index.js (hunk 1 above)
Comment (TheGoldenPatrik1) [reply app/sell/index.js:59]: My last comment might have been unclear, but we don't want manually refreshing to **stop** the periodic refresh, only to **reset** it. So maybe make this code a separate function:
```
        setTimeInterval(
            setInterval(() => {
                dispatch(getOrders)
            }, 60000)
        )
```
that gets called both within the useEffect hook and at the end of onRefresh?

**Review by TheGoldenPatrik1 — CHANGES_REQUESTED** [review]

File: app/sell/index.js (hunk 1 above)
Comment (lmillet3804) [reply app/sell/index.js:59]: Okay I fixed it now. I think I was just unclear about what clearInterval() did

**Review by TheGoldenPatrik1 — APPROVED** [review]

**Pull request merged**